src/Pyz/Yves/CheckoutPage/Theme/default/views/payment/payment.twig

```diff
@@ -79,8 +79,7 @@
                     } only %}
                         {% block fieldset %}
                             <div class="js-payment-method-{{ embed.index }} {% if embed.index != '1-0' %} is-hidden{% endif %}">
```

Comment:
I asked @danilmoro3 he said that the whole form should be hidden not only label (h2 tag) as you did.

Reply:
@matweew @danilmoro3 I didn't get your point.

There is only 1 form that contains all payment methods, we don't have separate form for each payment selection.

If you mean, when user choose Payone->Credit Card. It must hide form of previous selected payment method.

It is working in the same way currently. Let me know if you mean something else.

Reply:
> I asked @danilmoro3 he said that the whole form should be hidden not only label (h2 tag) as you did.

I tried to put condition on div instead of h2 tag, but there is JS event to hide/show div on payment radio selection.

If user select Payone->Credit Card payment, it removes **is-hidden** class from div and show the text. 

So either I do put condition on h2 tag or do not create div with condition.

Reply:
please choose the best solution from your point of view